tools/hangar/src/cli.test.ts

```diff
@@ -144,15 +151,53 @@ test.each(validWingFiles)(
 test.each(validWingFiles)(
   "wing test %s (--target sim)",
   async (wingFile) => {
-    const command = ["test"];
+    const args = ["test"];
     const testDir = path.join(tmpDir, `${wingFile}_sim`);
     fs.mkdirpSync(testDir);
 
-    await runWingCommand(command, path.join(validTestDir, wingFile), testDir);
+    await runWingCommand(
+      testDir,
+      path.join(validTestDir, wingFile),
+      args,
+      true
+    );
 
     // TODO snapshot .wsim contents
   },
   {
     timeout: 1000 * 30,
   }
 );
+
+test.each(invalidWingFiles)(
+  "wing test %s (--target sim) - invalid",
+  async (wingFile) => {
+    const args = ["test"];
+    const testDir = path.join(tmpDir, `invalid_${wingFile}_sim`);
+    fs.mkdirpSync(testDir);
+
+    const out = await runWingCommand(
+      testDir,
+      path.join(invalidTestDir, wingFile),
+      args,
+      false
+    );
+
+    const stderr = out.stderr;
+
+    const stderrSanitized = stderr
+      // Remove absolute paths
+      .replaceAll(invalidTestDir.replaceAll("\\", "/"), "<TEST_DIR>")
+      // Normalize line endings
+      .replaceAll("\r\n", "\n")
+      // For the error about a missing semicolon, remove the char number
+      // because it's usually the last char of the line which is 
+      // different due to windows line endings (`\r\n`)
+      .replace(/:\d+ \| ';' expected./g, ":0 | ';' expected.");
```

Comment:
As far as I could tell, this was the best of the worst ways to solve this.

Reply:
🙈 

Maybe we could normalize the line endings in the compiler (before processing the source file(s)) instead?

Reply:
I considered that but changing code in the compiler to account for this adds possible platform-specific behavior that doesn't actually help users and in fact may harm them (e.g. character position would actually be incorrect and not reflect reality)